spiceaidocs/content/en/reference/Spicepod/datasets.md

```diff
@@ -108,13 +108,35 @@ Optional. Accelerate queries to the dataset by caching data locally.
 
 ## `acceleration.enabled`
 
-Optional. Enable or disable acceleration.
+Enable or disable acceleration, defaults to `true`.
 
-## `acceleration.refresh`
+## `acceleration.engine`
 
-Optional. The interval to refresh the data for the dataset if the dataset type is overwrite.
+The acceleration engine to use, defaults to `arrow`. The following engines are supported:
 
-For `append` datasets, the refresh interval not used.
+  - `arrow` - Cache data as in-memory Apache Arrow Records.
```

Comment:
Remove the word Cache.

Use "Accelerated in-memory backed by Apache Arrow DataTables`